examples/ola-rdm.cpp

```diff
@@ -207,16 +207,22 @@ void DisplayHelpAndExit(const options &opts) {
   exit(ola::EXIT_USAGE);
 }
 
-
 /*
  * Dump the list of known pids
  */
 void DisplayPIDsAndExit(uint16_t manufacturer_id,
-                        const PidStoreHelper &pid_helper) {
+                        const PidStoreHelper &pid_helper,
+                        bool set_mode) {
+  vector<const ola::rdm::PidDescriptor *> pids;
+  pid_helper.SupportedPids(manufacturer_id, &pids);
+  // Remove PIDs that don't support the proper command class (GET with ola_rdm_get, SET with ola_rdm_set)
```

Comment:
This line is too long, they need to be 80 characters or less, so wrap it onto the next line.